src/components/Player/Pages/Trends/Trends.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { TrendGraph } from 'components/Visualizations';
+import { getPlayerMatches } from 'actions';
+import { playerMatches } from 'reducers';
+import { getCumulativeDataByField } from 'selectors';
+import { withRouter } from 'react-router';
+import FlatButton from 'material-ui/FlatButton';
+import { deSnake } from 'utility';
+import styles from './Trends.css';
+import trendNames from './trendNames';
+
+const selectTrend = (router, trendName, playerId) => {
+  router.push(`/players/${playerId}/trends/${trendName}`);
+};
+
+const Trend = ({ trendName = trendNames[0], columns, router, playerId }) => (
+  <div style={{ fontSize: 10 }}>
+    <div className={styles.buttonContainer}>
+      {trendNames.map((trend, index) => (
+        <FlatButton
+          onClick={() => selectTrend(router, trend, playerId)}
+          className={trendName === trend ? styles.selectedButton : styles.button}
+          key={index}
+        >
+          <span className={styles.buttonText}>{deSnake(trend)}</span>
```

Comment:
we should make an issue or mark with a TODO that deSnake will not work for localized strings.  We'll probably have to define the strings individually in the language files.